File: SISPU/client-app/src/app/classes/message.ts
import {Groupchat} from './groupchat'
import {User} from './user'

export class Message {

    id: string = "0";
    message_text: string = "";
    groupchat: Groupchat = new Groupchat();
    user: User = new User();
    date: Date = new Date();

    constructor()
    {
      this.id="";
      this.message_text= "";
      this.groupchat= new Groupchat();
      this.user= new User();
      this.date= new Date();

    }
}

File: SISPU/Models/App/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SISPU.Models
{

    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using Microsoft.AspNetCore.Identity;
    using Microsoft.AspNetCore.Identity.EntityFrameworkCore;

    public class Message
    {
        public int Id { get; set; }

        public string Text { get; set; }

        public GroupChat Chat { get; set; }

        public User User { get; set; }

        public DateTime Date { get; set; }

        public bool IsDeleted { get; set; }

        public Message()
        {
            IsDeleted = false;
        }

    }
}
